templates/definition/vehicle/iso15118.yaml

```diff
@@ -0,0 +1,68 @@
+template: iso15118
+products:
+  - description:
+      generic: ISO15118
+group: generic
+requirements:
+  description:
+    de: Nur unterstützt wenn das Fahrzeug den Ladestand (Soc) an die verbundene Wallbox übermitteln kann.
+    en: Only supported if the vehicle can provided the state of charge (Soc) to the connected wallbox.
+params:
+  - name: title
+  - name: icon
+    default: car
+    advanced: true
+  - name: capacity
+  - name: phases
```

Comment:
Alles ab phases wäre

      - preset: vehicle-identify

und in der Implementierung

    {{ include "vehicle-identify" . }}

Oder geht irgendein Parameter nicht?

Reply:
Ne, das passt. Passiert wenn man damit schon lange nichts mehr zu tun hatte, auch wenn ich es damals gebaut hatte ;)